types/timeDefinitions.tsx

```diff
@@ -10,8 +10,12 @@ export const timeDefinitions = [
 export const getTimeDefinitionValues = (): Array<string> =>
   timeDefinitions.map((item) => item.displayName);
 
-export const getTimeDefinitionValue = (displayName: string): string =>
-  timeDefinitions.find((item) => item.displayName === displayName).value;
+export const getTimeDefinitionValue = (displayName: string): string | null => {
```

Comment:
Please remove the `|| null` as the places this function is used it expects `string | undefined`.

Reply:
Type null is what we get from the server, so VsmObject.tsx should have had this typing already. I will add it. Passing null for this changed input field is good to indicate to the backend that changes has been made by the user.